### PR TITLE
Test fixes

### PR DESCRIPTION
**Pull request by benspaulding**
Prior to this much of the test suite failed due to str/bytes issues and division. Now most of it is passing.

### CODE REVIEW

File: kitsune/kbforums/models.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ class Meta:
     @property
     def last_page(self):
         """Returns the page number for the last post."""
-        return self.replies / kbforums.POSTS_PER_PAGE + 1
+        return self.replies // kbforums.POSTS_PER_PAGE + 1
 
     def __unicode__(self):
```
Comment (akatsoulas) [inline kitsune/kbforums/models.py]: Not relevant to this fix. Mainly a note that we need to update these methods too.

**Comment by benspaulding** [comment]
@akatsoulas CircleCI failed because of a package upgrade I did and I need some guidance on how you want it fixed.

We use [django-authority](https://pypi.org/project/django-authority/), installed from PyPI. There is a fix we need [e3e4cd7](https://github.com/jazzband/django-authority/commit/e3e4cd713ef084009a98b035109326cd168a7e47) that is not in a release yet. Normally we would just install using the tarball on GitHub:

```
https://github.com/jazzband/django-authority/archive/e3e4cd713ef084009a98b035109326cd168a7e47.tar.gz#egg=django-authority
```

But their setup.py requires a proper git repository to install. We cannot install from a repo though because hashes don’t work for them.

```
git+https://github.com/jazzband/django-authority.git@e3e4cd713ef084009a98b035109326cd168a7e47#egg=django-authority
```

Should we contact them about making a release, or do you want me to fork the repo and fix up the setup.py to work for us?

**Comment by akatsoulas** [comment]
@benspaulding I just had a talk with @safwanrahman and he just created a release in Github [0]. Hopefully this will soon land to pypi as he needs to coordinate with the project owner. 

Thanks @safwanrahman for pushing this so fast!

[0] https://github.com/jazzband/django-authority/releases/tag/0.14

**Comment by benspaulding** [comment]
> … I just had a talk with @safwanrahman and he just created a release in Github [0]. Hopefully this will soon land to pypi as he needs to coordinate with the project owner first.
> 
> Thanks @safwanrahman for pushing this so fast!

Yes, thank you @safwanrahman! Do you have any idea if @jazzband is planning to make that release soon? Or is there someone I could contact about it?

**Comment by safwanrahman** [comment]
@benspaulding I have talked with one of the people of jazzband personally and I hope that it will be sorted out soon. As they have just come back from long all hands of mozilla!
https://github.com/jazzband-roadies/help/issues/183

**Pull request merged**